src/components/side_panel/pivot/pivot_side_panel/pivot_spreadsheet_side_panel/pivot_spreadsheet_side_panel.xml

```diff
@@ -2,7 +2,7 @@
   <t t-name="o-spreadsheet-PivotSpreadsheetSidePanel">
     <div class="d-flex flex-column h-100 justify-content-between overflow-hidden">
       <div class="h-100 overflow-auto">
-        <PivotTitleSection pivotId="props.pivotId"/>
+        <PivotTitleSection pivotId="props.pivotId" flipAxis.bind="flipAxis"/>
```

Comment:
How do you feel about passing the sotre as props rather than the flipAxis callback ? That was the flipAxis method would not need to be duplicated in odoo.